tests/js/utils.test.js

```diff
@@ -201,4 +203,125 @@ describe( 'test utilities', () => {
 			);
 		} );
 	} );
+
+	describe( 'provideWidgetRegistrations', () => {
+		let registry;
+
+		beforeEach( () => {
+			registry = createTestRegistry();
+		} );
+
+		it( 'should register default widget areas and widgets without extra data', () => {
+			provideWidgetRegistrations( registry );
+
+			const expectedRegistrations = [
+				{ key: 'mainDashboardKeyMetricsPrimary', type: 'widgetArea' },
+				{ key: 'mainDashboardTrafficPrimary', type: 'widgetArea' },
+				{ key: 'keyMetricsSetupCTA', type: 'widget' },
+			];
+
+			expectedRegistrations.forEach( ( { key, type } ) => {
+				if ( type === 'widgetArea' ) {
+					expect(
+						registry
+							.select( CORE_WIDGETS )
+							.isWidgetAreaRegistered( key )
+					).toBe( true );
+				} else if ( type === 'widget' ) {
+					expect(
+						registry
+							.select( CORE_WIDGETS )
+							.isWidgetRegistered( key )
+					).toBe( true );
+				}
+			} );
+		} );
```

Comment:
@benbowler, i mean that we need to write it using [it.each](https://jestjs.io/docs/api#testeachtablename-fn-timeout):

```js
it.each( [
    [ 'mainDashboardKeyMetricsPrimary' ],
    [ 'mainDashboardTrafficPrimary' ],
] )( 'should register %s widget area without extra data', ( widgetArea ) => {
    provideWidgetRegistrations( registry );
    expect( registry.select( CORE_WIDGETS ).isWidgetAreaRegistered( key ) ).toBe( true );
} );

it( 'should register default widgets without extra data', () => {
    provideWidgetRegistrations( registry );
	expect( registry.select( CORE_WIDGETS ).isWidgetRegistered( 'keyMetricsSetupCTA' ) ).toBe( true );
}
```

Reply:
Or we can even combine them using the selector name as the second item of the table:

```js
it.each( [
    [ 'mainDashboardKeyMetricsPrimary','isWidgetAreaRegistered' ],
    [ 'mainDashboardTrafficPrimary', 'isWidgetAreaRegistered' ],
    [ 'keyMetricsSetupCTA', 'isWidgetRegistered' ],
] )( 'should register %s without extra data', ( key, selector ) => {
    provideWidgetRegistrations( registry );
    expect( registry.select( CORE_WIDGETS )[ selector ]( key ) ).toBe( true );
} );
```